br/point-in-time-recovery.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Point-in-Time Recovery
+summary: Learn the design, capabilities, and architecture of Point-in-Time Recovery (PITR).
+---
+
+# Point-in-Time Recovery
+
+Point-in-Time Recovery (PITR) allows you to restore a snapshot of a TiDB cluster to a new cluster from any given time point in the past. In v6.2.0, TiDB introduces PITR in [Backup & Restore](/br/backup-and-restore-overview.md) (BR).
+
+You can use PITR to meet the following business requirements:
+
+- Reduce the RPO of disaster recovery to less than 20 minutes.
+- Handle the cases of incorrect writes from applications by rolling back data to before the error event.
+- Perform history data auditing to meet the requirements of laws and regulations.
+
+This document introduces the design, capabilities, and architecture of PITR. If you need to learn how to use PITR, refer to [PITR Usage Scenarios](/br/pitr-usage.md).
+
+## Use PITR in your business
+
+[BR](/br/backup-and-restore-overview.md) is the tool for using the PITR feature. With BR, you can perform all operations of PITR, including data backup (snapshot backup and log backup), one-click restoration to a specified time point, and backup data management.
+
+The following are the procedures of using PITR in your business:
+
+![Point-in-Time Recovery](/media/br/pitr-usage.png)
```

Comment:
There are some errors in the diagram:
- 'these data' should be 'the following data'
- 'logs backups' should be 'log backups'
- 'PiTR' should 'PITR'

Reply:
fixed in https://github.com/pingcap/docs-cn/pull/10841